operator/src/test/java/org/keycloak/operator/testsuite/integration/KeycloakDeploymentTest.java

```diff
@@ -770,6 +773,86 @@ public void testApplyingResourcesDefaultValues() {
         assertThat(limits.get("memory")).isEqualTo(config.keycloak().resources().limits().memory());
     }
 
+    @Test
+    public void testTracingSpec() {
+        var kc = getTestKeycloakDeployment(true);
+        kc.getSpec().setStartOptimized(false);
+
+        var featureSpec = new FeatureSpec();
+        featureSpec.setEnabledFeatures(List.of("opentelemetry"));
+        kc.getSpec().setFeatureSpec(featureSpec);
```

Comment:
We shouldn't need this if #35756 gets merged first.

Reply:
Yes, we will not need this after getting that merged.